src/elf.rs

```diff
@@ -7,9 +7,7 @@
 
 use goblin;
 use failure::Error;
-use {artifact, Artifact, Decl, Ctx, ImportKind};
-use artifact::{LinkAndDecl, Reloc};
-use target::make_ctx;
+use crate::{artifact::{self, LinkAndDecl, Reloc}, Artifact, Decl, Ctx, ImportKind, target::make_ctx};
```

Comment:
I prefer keeping these on separate lines. Moving the `artifact::self` to the other line is fine though.

Reply:
I like these sort of declarations because when run through `rustfmt` they are rendered like:
`use crate::{
    artifact::{self, LinkAndDecl, Reloc},
    target::make_ctx,
    Artifact, Ctx, Decl, ImportKind,
};
`
But I didn't do any work to space them out that way by hand in this PR, because I was hoping to do `rustfmt` separately once I got feedback in #62.